monitoring/grafana/dashboards/tests/__init__.py

```diff
@@ -171,7 +171,8 @@ def run_promtool(self):
 
         for variable, value in self.variables.items():
             expr = self.promql_expr_test.expr
-            new_expr = re.sub(r'\${0}'.format(variable), str(value), expr)
+            regex = fr'\${variable}(?=\W)'
+            new_expr = re.sub(regex, fr'{str(value)}', expr)
```

Comment:
f-strings take care of converting objects to string representations:
```suggestion
            new_expr = re.sub(regex, fr'{value}', expr)
```

Reply:
true